share/spice/quote_of_the_day/quote_of_the_day.js

```diff
@@ -0,0 +1,35 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_quote_of_the_day = function(api_result) {
+
+        if (!api_result || api_result.error) {
+            return Spice.failed('quote_of_the_day');
+        }
+
+        Spice.add({
+            id: "quote_of_the_day",
+
+            name: "Quote of the Day",
+            data: api_result,
+            meta: {
+                sourceName: "They Said So",
+                sourceUrl: "https://theysaidso.com/quote/"+api_result.contents.quotes[0].id
```

Comment:
If we set `signal` to `"high"` it should *(hopefully)* override the Wikipedia Fathead that is currently taking priority on `quote of the day`.